observationtree/src/main/scala/explore/observationtree/TargetObsList.scala

```diff
@@ -269,76 +272,83 @@ object TargetObsList {
-                              )(ExploreStyles.SelectedObsTreeGroup)
-                              .getOrElse(ExploreStyles.UnselectedObsTreeGroup)
+                      AppCtx.withCtx { ctx =>
+                        Droppable(target.id.toString) { case (provided, snapshot) =>
+                          <.div(
+                            provided.innerRef,
+                            provided.droppableProps,
+                            getListStyle(snapshot.isDraggingOver)
                           )(
-                            ^.cursor.pointer,
-                            ^.onClick --> props.focused.set(FocusedTarget(targetId).some).runAsyncCB
-                          )(
-                            <.span(ExploreStyles.ObsTreeGroupHeader)(
-                              <.span(
-                                opIcon,
-                                target.name.value,
-                                ExploreStyles.TargetLabelTitle
-                              ),
-                              Button(
-                                size = Small,
-                                compact = true,
-                                clazz = ExploreStyles.DeleteButton |+| ExploreStyles.JustifyRight,
-                                onClickE = (e: ReactMouseEvent, _: ButtonProps) =>
-                                  e.stopPropagationCB *>
-                                    deleteTarget(targetId, undoCtx.setter, focusOnDelete)
-                              )(
-                                Icons.Delete
-                                  .size(Small)
-                                  .fitted(true)
-                                  .clazz(ExploreStyles.TrashIcon)
+                            Segment(
+                              vertical = true,
+                              clazz = ExploreStyles.ObsTreeGroup |+| Option
+                                .when(
+                                  memberObsSelected || props.focused.get
+                                    .exists(_ === focusedTarget)
+                                )(ExploreStyles.SelectedObsTreeGroup)
+                                .getOrElse(ExploreStyles.UnselectedObsTreeGroup)
+                            )(
+                              ^.cursor.pointer,
+                              ^.onClick --> props.focused
+                                .set(focusedTarget.some)
+                                .runAsyncCB
+                            )(
+                              <.span(ExploreStyles.ObsTreeGroupHeader)(
+                                <.a(
+                                  ^.href := ctx.pageUrl(AppTab.Targets, focusedTarget.some),
+                                  opIcon,
+                                  target.name.value,
+                                  ExploreStyles.TargetLabelTitle,
+                                  ^.onClick ==> {(e: ReactEvent) => e.preventDefaultCB}
```

Comment:
I'm not sure I'm following this. What is having onClick prevent the default action on an anchor tag preventing?

Reply:
If you don't prevent it will go to the correct url but via a page reload rather than an internal navegation

Reply:
👍